### ISSUE TITLE
Exception on sending mail

### ISSUE THREAD

**Issue by midcielab** [midcielab/feeds-client#1]
**Symptom**

Sending mail may fail randomly with below exception

```java
javax.mail.MessagingException: IOException while sending message;
  nested exception is:
        javax.activation.UnsupportedDataTypeException: no object DCH for MIME type text/plain; charset=UTF-8
        at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1365)
        at javax.mail.Transport.send0(Transport.java:255)
        at javax.mail.Transport.send(Transport.java:124)
        at com.midcielab.utility.SmtpUtility.sendMail(SmtpUtility.java:43)
        at com.midcielab.handler.FeedsHandler.performAction(FeedsHandler.java:94)
        at com.midcielab.handler.FeedsHandler.process(FeedsHandler.java:41)
        at com.midcielab.FeedsClient.main(FeedsClient.java:8)
Caused by: javax.activation.UnsupportedDataTypeException: no object DCH for MIME type text/plain; charset=UTF-8
        at javax.activation.ObjectDataContentHandler.writeTo(DataHandler.java:885)
        at javax.activation.DataHandler.writeTo(DataHandler.java:316)
        at javax.mail.internet.MimeBodyPart.writeTo(MimeBodyPart.java:1694)
        at javax.mail.internet.MimeMessage.writeTo(MimeMessage.java:1913)
        at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1315)
```

**Comment by midcielab** [comment]
Debug with -Djavax.activation.debug=true flag, trace as below

```java
MailcapCommandMap: load HOME
new MailcapFile: file /home/midcielab/.mailcap
MailcapCommandMap: load SYS
new MailcapFile: file /home/midcielab/graalvm/lib/mailcap
MailcapCommandMap: load JAR
MailcapCommandMap: getResources
MailcapCommandMap: URL jar:file:/home/midcielab/test/feedsclient-1.0.0-shaded.jar!/META-INF/mailcap
new MailcapFile: InputStream
parse: text/plain;;             x-java-content-handler=com.sun.mail.handlers.text_plain
  Type: text/plain
    Command: content-handler, Class: com.sun.mail.handlers.text_plain
parse: text/html;;              x-java-content-handler=com.sun.mail.handlers.text_html
  Type: text/html
    Command: content-handler, Class: com.sun.mail.handlers.text_html
parse: text/xml;;               x-java-content-handler=com.sun.mail.handlers.text_xml
  Type: text/xml
    Command: content-handler, Class: com.sun.mail.handlers.text_xml
parse: multipart/*;;            x-java-content-handler=com.sun.mail.handlers.multipart_mixed; x-java-fallback-entry=true
  Type: multipart/*
    Command: content-handler, Class: com.sun.mail.handlers.multipart_mixed
parse: message/rfc822;; x-java-content-handler=com.sun.mail.handlers.message_rfc822
  Type: message/rfc822
    Command: content-handler, Class: com.sun.mail.handlers.message_rfc822
MailcapCommandMap: successfully loaded mailcap file from URL: jar:file:/home/midcielab/test/feedsclient-1.0.0-shaded.jar!/META-INF/mailcap
MailcapCommandMap: load DEF
new MailcapFile: InputStream
parse: image/gif;;              x-java-view=com.sun.activation.viewers.ImageViewer
  Type: image/gif
    Command: view, Class: com.sun.activation.viewers.ImageViewer
parse: image/jpeg;;             x-java-view=com.sun.activation.viewers.ImageViewer
  Type: image/jpeg
    Command: view, Class: com.sun.activation.viewers.ImageViewer
parse: text/*;;         x-java-view=com.sun.activation.viewers.TextViewer
  Type: text/*
    Command: view, Class: com.sun.activation.viewers.TextViewer
parse: text/*;;         x-java-edit=com.sun.activation.viewers.TextEditor
  Type: text/*
    Command: edit, Class: com.sun.activation.viewers.TextEditor
Merging commands for type text/*
MailcapCommandMap: successfully loaded mailcap file: /META-INF/mailcap.default
MailcapCommandMap: createDataContentHandler for text/plain
  search DB #1
    got content-handler
      class com.sun.mail.handlers.text_plain
Can't load DCH com.sun.mail.handlers.text_plain; Exception: java.lang.ClassNotFoundException: com.sun.mail.handlers.text_plain
  search DB #2
  search fallback DB #1
  search fallback DB #2
javax.mail.MessagingException: IOException while sending message;
  nested exception is:
        javax.activation.UnsupportedDataTypeException: no object DCH for MIME type text/plain; charset=UTF-8
        at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1365)
        at javax.mail.Transport.send0(Transport.java:255)
        at javax.mail.Transport.send(Transport.java:124)
        at com.midcielab.utility.SmtpUtility.sendMail(SmtpUtility.java:43)
        at com.midcielab.handler.FeedsHandler.performAction(FeedsHandler.java:94)
        at com.midcielab.handler.FeedsHandler.process(FeedsHandler.java:41)
        at com.midcielab.FeedsClient.main(FeedsClient.java:8)
Caused by: javax.activation.UnsupportedDataTypeException: no object DCH for MIME type text/plain; charset=UTF-8
        at javax.activation.ObjectDataContentHandler.writeTo(DataHandler.java:885)
        at javax.activation.DataHandler.writeTo(DataHandler.java:316)
        at javax.mail.internet.MimeBodyPart.writeTo(MimeBodyPart.java:1694)
        at javax.mail.internet.MimeMessage.writeTo(MimeMessage.java:1913)
        at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1315)
```

It seems like a classloading problem.

**Comment by midcielab** [comment]
Fix with v1.0.1, issue close.
